lib/tesla/middleware/method_override.ex

```diff
@@ -1,9 +1,9 @@
 defmodule Tesla.Middleware.MethodOverride do
   @moduledoc """
-  Middleware that adds X-Http-Method-Override header with original request
+  Middleware that adds `X-Http-Method-Override` header with original request
```

Comment:
I think it would be better to downcase the example as in the code.

Suggestion

> Changes the request method to POST and adds `x-http-method-override` header with the original request method

Reply:
Based on https://opensocial.github.io/spec/2.5.1/Core-API-Server.xml#rfc.section.2.1.1.1 via https://en.wikipedia.org/wiki/List_of_HTTP_header_fields,
`X-HTTP-Method-Override` seems to be the right name with case.